editor/src/components/editor/store/store-deep-equality-instances.ts

```diff
@@ -1578,12 +1578,9 @@ export function SpecialSizeMeasurementsKeepDeepEquality(): KeepDeepEqualityCall<
     const clientHeightResult = oldSize.clientHeight === newSize.clientHeight
     const parentFlexDirectionResult = oldSize.parentFlexDirection === newSize.parentFlexDirection
     const parentJustifyContentEquals = oldSize.parentJustifyContent === newSize.parentJustifyContent
-    const parentFlexGapEquals = NumberKeepDeepEquality(oldSize.parentFlexGap, newSize.parentFlexGap)
+    const parentFlexGapEquals = oldSize.parentFlexGap === newSize.parentFlexGap
     const parentPaddingEquals = SidesKeepDeepEquality(oldSize.parentPadding, newSize.parentPadding)
-    const parentHugsOnMainAxisEquals = BooleanKeepDeepEquality(
-      oldSize.parentHugsOnMainAxis,
-      newSize.parentHugsOnMainAxis,
-    )
+    const parentHugsOnMainAxisEquals = oldSize.parentHugsOnMainAxis === newSize.parentHugsOnMainAxis
```

Comment:
These `===` are a little brittle because if we'd change `parentFlexGap` or `parentHugsOnMainAxis` for any reason, we won't get any type errors, so I'd keep the `*DeepEquality` calls and add the `.areEqual` prop instead